include/votca/xtp/gw.h

```diff
@@ -111,7 +111,12 @@ class GW {
         : _gw_level(gw_level), _offset(offset), _sigma_c_func(sigma){};
     std::pair<double, double> operator()(double frequency) const {
       std::pair<double, double> value;
-      value.first = value(frequency) value.second = deriv(frequency);
+      value.first =
+          _sigma_c_func.CalcCorrelationDiagElement(_gw_level, frequency);
+      value.second = _sigma_c_func.CalcCorrelationDiagElementDerivative(
+          _gw_level, frequency);
+      value.first += (_offset - frequency);
+      value.second -= 1.0;
```

Comment:
why this? The old stuff was missing a `;` but otherwise it should just call the individual functions.

Reply:
I didn't realize that it had failed thanks to double use of `value`. Changed the return variable name.